public/index.php

```diff
@@ -7,6 +7,11 @@
  */
 chdir(dirname(__DIR__));
 
+// For Docker:
+if (file_exists('/code')) {
```

Comment:
Why are these lines of code needed? I assume public/index.php is located in /code/public/index.php? So then it should work without this.

Reply:
Well....to get the file sharing with the other container working properly, I created three volumes: /code, /public and /data.
/code contains symlinks to the other volumes. So index.php is located in /public/index.php.
It's not the most neat solution, but it works :)

Reply:
I think you removed the /code volume since it was not needed?
The public volume you can mount to /code/public and the data volume you can mount to /code/data

Reply:
I indeed removed /code as a volume that was shared. Mmmmhm. Perhaps I could glue it back together

Reply:
I fixed it!